zebrad/tests/common/lightwalletd/wallet_grpc_test.rs

```diff
@@ -337,6 +338,74 @@ pub async fn run() -> Result<()> {
         *zebra_test::vectors::SAPLING_TREESTATE_MAINNET_419201_STRING
     );
 
+    // Call `z_getsubtreesbyindex` separately for
+
+    // ... Sapling.
+    let mut subtrees = rpc_client
+        .get_subtree_roots(GetSubtreeRootsArg {
+            start_index: 0u32,
+            shielded_protocol: ShieldedProtocol::Sapling.into(),
+            max_entries: 2u32,
+        })
+        .await?
+        .into_inner();
+
+    let mut counter = 0;
+    while let Some(_subtree) = subtrees.message().await? {
+        match counter {
+            0 => {
+                // TODO: add asserts that check:
+                // 1. The subtree hash.
+                // 2. The completing block hash.
+                // 3. The completing block height.
```

Comment:
@teor2345 do you think it's worth adding and checking concrete vectors here?

My answer would be yes, but I haven't added them yet because I have to obtain them first.

Reply:
A big list of test vectors will be added in PR #7515, but feel free to add the first subtree here if you'd like. It's ok to just check mainnet and to use Zebra to generate the test vectors. (Ticket #7450 checks that Zebra matches `zcashd` for the first subtree.)

Reply:
OK. I'm going to add two subtrees for Sapling and two for Orchard to test the RPC returns two consecutive subtrees.

Reply:
Done in [06d4484](https://github.com/ZcashFoundation/zebra/pull/7521/commits/06d448425fbc54ec127ec37c4e8931e4883a899c).